src/Elewant/AppBundle/Resources/views/Herd/tending.html.twig

```diff
@@ -1,11 +1,22 @@
 {% extends 'ElewantAppBundle::Layout/base.html.twig' %}
 {% trans_default_domain "herd" %}
 
-{% block content %}
+{% block javascripts %}
+    <script type="application/javascript" src="{{ asset('build/js/rename-herd.js') }}"></script>
+    <script type="application/javascript">
+        var herdNameHolder     = $("#herd_tending");
+        var renameHerdFunction = new RenameHerd(herdNameHolder);
 
-    <section id="herd_tending">
+        renameHerdFunction.initialize();
+    </script>
```

Comment:
Can you also place this initialization code into a global javascript file?

Also, when the "class" (function in js) has a method with the same name (`RenameHerd`), it is run as a constructor. So maybe call the `initialize()` method from there? Or move the actual code to the constructor?